tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.py

```diff
@@ -0,0 +1,25 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def displacement(
+        x, length, height, bending_moment, bulk_modulus, shear_modulus):
+    youngs_modulus = 9 * bulk_modulus * shear_modulus / \
```

Comment:
9 -> 9.0, and similarly everywhere you multiply a hard-coded number by a double, to avoid oddities with e.g. integer vs. double division in some old python versions.